src/middle/Mir.ml

```diff
@@ -200,3 +200,5 @@ type internal_fn =
   | FnPrint
   | FnReject
 [@@deriving sexp]
+
+let proportional_to_distribution_infix = "_propto"
```

Comment:
Is this the syntax we are now going for for the unnormalized distributions?

Reply:
So I needed a way when translating the AST to the MIR to mark a distribution as propto (Bob's preferred keyword). We thought it would be a good idea to just include that in the function name and allow users to use it too. It shows up in code gen because we have to remove it.

Reply:
Ah, re: the actual word choice - I don't care at all. Bob had a strong opinion in #63. You guys can battle it out there :P

Reply:
OK! So now this propto could be in function names form the MIR onwards?
I wonder if we should patch the type checking so it can also deal with it seeing that some optimizations will use the type checking on function names appearing in the MIR. Then it'll automatically also will become available in the language.

Reply:
Yeah, that's right. That sounds good to me, and happy to switch the actual word choice to whatever.